registry/lombard/calldata-lbtc-redeem-mainnet.json

```diff
@@ -0,0 +1,1494 @@
+            {
+              "name": "value",
+              "type": "uint256",
+              "internalType": "uint256",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [
+            {
+              "name": "",
+              "type": "bool",
+              "internalType": "bool",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "transferOperatorRole",
+          "inputs": [
+            {
+              "name": "newOperator",
+              "type": "address",
+              "internalType": "address",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "transferOwnership",
+          "inputs": [
+            {
+              "name": "newOwner",
+              "type": "address",
+              "internalType": "address",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "transferPauserRole",
+          "inputs": [
+            {
+              "name": "newPauser",
+              "type": "address",
+              "internalType": "address",
+              "components": null,
+              "indexed": null,
+              "unit": null
+            }
+          ],
+          "outputs": [],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        },
+        {
+          "type": "function",
+          "name": "unpause",
+          "inputs": [],
+          "outputs": [],
+          "stateMutability": "nonpayable",
+          "constant": null,
+          "payable": null,
+          "gas": null,
+          "signature": null
+        }
+      ]
+    }
+  },
+  "metadata": {
+    "owner": "Lombard",
+    "info": {
+      "legalName": "Lombard Finance",
+      "url": "https://www.lombard.finance/"
+    }
+  },
+  "display": {
+    "formats": {
+      "redeem(bytes,uint256)": {
+        "intent": "Redeem LBTC",
+        "fields": [
+          {
+            "label": "Script Pubkey",
+            "format": "raw",
```

Comment:
Not 100% sure that this is the correct format for a `bytes` input?

Reply:
It looks good to me, It will display the scriptPubKey as a hexadecimal string.